content/guides/component-testing/introduction.md

```diff
@@ -114,10 +114,11 @@ look like this:
 
 ```js
 {
-  testFiles: "cypress/integration/*.spec.js",
+  e2e: {
```

Comment:
Shouldn't this file actually be deleted? See https://github.com/cypress-io/cypress-documentation/blob/unification/netlify.toml#L56-L60

Reply:
Yeah you're right. It's still in `unification`. IDK if a previous PR was to handle that. I'll pull it from here.